<ui version="4.0" >
 <class>bandwidth_dlg</class>
 <widget class="QDialog" name="bandwidth_dlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Bandwidth allocation</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="lblTitle" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="limit_lbl" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="kb_lbl" >
       <property name="text" >
        <string>KiB/s</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="bandwidthSlider" >
     <property name="minimum" >
      <number>0</number>
     </property>
     <property name="maximum" >
      <number>1000</number>
     </property>
     <property name="sliderPosition" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>bandwidth_dlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>212</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel" >
     <x>221</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
